meetupapi/src/main/java/com/gdgistanbul/api/authenticator/RequestInterceptor.kt

```diff
@@ -0,0 +1,30 @@
+package com.gdgistanbul.api.authenticator
+
+import android.content.SharedPreferences
+import com.gdgistanbul.extension.loginAdapter
+import com.squareup.moshi.Moshi
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class RequestInterceptor(
```

Comment:
I think TokenAuthenticator was a better approach.  Interceptor doesn't refresh the token in the case of 401.

Reply:
While working on it, I created a circular dependency and decided to go with that way. As thinking again, I figured it was my fault. I'll go over it again.

Reply:
I've just researched Authenticator but we have a problem. OkHttp interceptors doesn't support Coroutines so we have to handle this in another way.

Reply:
I still think the Authenticator is a better solution. The authenticator can retry requests and refresh tokens. The second `getAccessToken` method in the api is deleted. Now there is no way to refresh the token once it is expired.

Reply:
Ok it's done.